packages/react-dev-utils/README.md

```diff
@@ -166,6 +166,43 @@ compiler.plugin('done', function(stats) {
 });
 ```
 
+#### `formatter(results: Object): string`
+
+This is an eslint formatter that takes the result generated from eslint and formats the output string
+
+```js
+const formatter = require('react-dev-utils/formatter');
+const webpack = require('webpack');
+
+module: {
+   strictExportPresence: true,
+   rules: [
+     { parser: { requireEnsure: false } },
+     {
+        test: /\.(js|jsx)$/,
+        enforce: 'pre',
+        use: [
+          {
+            // @remove-on-eject-begin
```

Comment:
Nobody knows what `@remove-on-eject-begin` means. This is our internal thing. Please remove it from the doc.